docs/reference/verification-result-version.md

```diff
@@ -84,6 +107,119 @@ definitions:
 }
 ```
 
+## 0.2.0
```

Comment:
Suggestion to add some context here around which policy produces which version of response, and the difference between them. If we already have this information captured, a link would be nice. thanks!

Reply:
Good suggestion, added a new section `Verification Response Versions and Policy Providers` to explain it.

Reply:
thankyou binbin. looks good to me